docs/user/integration/dynatrace/README.md

```diff
@@ -82,21 +84,21 @@ As a result, you see data arriving in your environment, advanced Kubernetes moni
 
 Next, you set up the ingestion of custom span and Istio span data, and, optionally, custom metrics based on OTLP.
 
-### Create Access Token
+### Create Access Token and Secret
 
-To push custom metrics and spans to Dynatrace, set up an [API Token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
+To push custom metrics and spans to Dynatrace, set up [dataIngestToken](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
```

Comment:
Why not format instructions as numbered steps, just like in the previous section? 
```suggestion
1. To push custom metrics and spans to Dynatrace, set up a [dataIngestToken](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
```